content/download/fedora.adoc

```diff
@@ -4,14 +4,24 @@ iconhtml = "<div class='fl-fedora'></div>"
 weight = 20
 +++
 
-=== 4.0.2 Stable Release
-KiCad 4.0.2 is available in the Fedora 23 repo.
+=== Stable Release
+In general, latest Fedora releases ship the stable versions KiCad as they are
+released, with a delay of around a week that the package spends in the testing
```

Comment:
I think we should the info about this delay. It is not likely to be accurate nor really that usefull anyways.